ktor-http/src/io/ktor/http/ContentTypes.kt

```diff
@@ -184,6 +184,7 @@ class ContentType private constructor(val contentType: String, val contentSubtyp
         val Any = ContentType("text", "*")
         val Plain = ContentType("text", "plain")
         val CSS = ContentType("text", "css")
+        val Csv = ContentType("text", "csv")
```

Comment:
I'd say this need to be `CSV` as we already have abbreviations uppercased

Reply:
Fixed